### ISSUE TITLE
keep printing out `etcdserver: requested snapshot created at` every second

### ISSUE THREAD

**Issue by yichengq** [coreos/etcd#3778]
This happens when one follower is dead, and brings limited information:

```
2015-10-28 22:07:53.154154 I | etcdserver: requested snapshot created at
1776711
2015-10-28 22:07:53.654527 I | etcdserver: requested snapshot created at
1776711
2015-10-28 22:07:54.154223 I | etcdserver: requested snapshot created at
1776711
```

The problem is two:
1. it keeps requesting every second, even if the follower is dead
2. it doesn't display the result of these requested snapshots.

from #3775

**Comment by jonboulle** [comment]
We need to stop attempting to send the snapshot until the peer comes back online.

**Comment by xiang90** [comment]
This really needs to be solved soon. The goal of 2.3 is to make snapshot working properly.

**Comment by xiang90** [comment]
closed via #3976
